acceptance-tests/src/acceptance-test/java/tech/pegasys/teku/test/acceptance/BlsToExecutionChangeAcceptanceTest.java

```diff
@@ -40,14 +41,19 @@ void shouldUpdateWithdrawalCredentials() throws Exception {
 
     final TekuNode primaryNode = createPrimaryNode(executionAddress, capellaActivationEpoch);
     primaryNode.start();
+    primaryNode.startEventListener(List.of(EventType.bls_to_execution_change));
 
     final TekuNode lateJoiningNode =
         createLateJoiningNode(capellaActivationEpoch, primaryNode, primaryNode.getGenesisTime());
     lateJoiningNode.start();
     lateJoiningNode.waitUntilInSyncWith(primaryNode);
+    lateJoiningNode.startEventListener(List.of(EventType.bls_to_execution_change));
```

Comment:
nit: It would be much nicer if we adjusted this method to just use varargs instead of requiring the AT to create the list wrapper.